src/stepFunctions/commands/visualizeStateMachine/aslVisualizationCDKManager.ts

```diff
@@ -0,0 +1,105 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+
+import { AslVisualizationCDK } from './aslVisualizationCDK'
+import { AbstractAslVisualizationManager } from './abstractAslVisualizationManager'
+import { ConstructNode } from '../../../cdk/explorer/nodes/constructNode'
+import { getLogger } from '../../../shared/logger'
+import { StateMachineGraphCache } from '../../utils'
+
+export class AslVisualizationCDKManager extends AbstractAslVisualizationManager {
+    protected readonly name: string = 'AslVisualizationCDKManager'
+    protected readonly managedVisualizations: Map<string, Map<string, AslVisualizationCDK>> = new Map<
+        string,
+        Map<string, AslVisualizationCDK>
+    >()
+
+    public constructor(extensionContext: vscode.ExtensionContext) {
+        super(extensionContext)
+    }
+
+    //vscode.Memento passed in to update StateMachineGraphCache on each visualization
+    public async visualizeStateMachine(
+        globalStorage: vscode.Memento,
+        node: ConstructNode
+    ): Promise<vscode.WebviewPanel | undefined> {
+        const logger = getLogger()
+        const cache = new StateMachineGraphCache()
```

Comment:
Creating a new cache every time you call this function seems counter to having a cache. Should this be persisted somewhere?